src/sardana/pool/poolcontrollers/DiscretePseudoMotorController.py

```diff
@@ -119,10 +130,20 @@ def CalcPseudo(self, axis, physical_pos, curr_pseudo_pos):
             raise Exception("Bad configuration on axis attributes.")
 
     def CalcPhysical(self, axis, pseudo_pos, curr_physical_pos):
+
+        if self._configuration is not None:
+            positions = self._positions_cfg
+            calibration = self._calibration_cfg
+            labels = self._labels_cfg
+        else:
+            # TODO: Remove when we do not support more
+            print('holaaaa', self._labels)
```

Comment:
Could you please remove this comment? :)

Reply:
Ups! Yes of course.